values-factory.yaml

```diff
@@ -12,29 +12,6 @@ clusterGroup:
   - manuela-factory-ml-workspace
 
   subscriptions:
-  - name: opendatahub-operator
-    source: community-operators
-    csv: opendatahub-operator.v1.1.0
-
-  - name: seldon-operator
-    namespace: manuela-stormshift-messaging
-    source: community-operators
-    csv: seldon-operator.v1.12.0
-
-  - name: amq-streams
-    namespace: manuela-stormshift-messaging
-    channel: amq-streams-2.x
-    csv: amqstreams.v2.0.1-0
-
-  - name: amq-broker-rhel8
-    namespace: manuela-stormshift-messaging
-    channel: 7.x
-    csv: amq-broker-operator.v7.8.1-opr-3
-
-  - name: red-hat-camel-k
-    namespace: manuela-stormshift-messaging
-    channel: 1.6.x
-    csv: red-hat-camel-k-operator.v1.6.3
```

Comment:
I think we should try and be less invasive in this change. And leave the common "good" defaults in values-factory.yaml here and only put the specific overrides (I think it is only amq-streams in this case?) in the per ocp-version override files. This way it will be a lot clearer as to why they are needed and what is different. So I'd leave all operators in values-factory.yaml and then just the amq-streams overrides as needed